### ISSUE TITLE
needs form labels

### ISSUE THREAD

**Issue by sionleroux** [sinisterstuf/icfbudapest.org#1]
lots of Windows browsers don't show the placeholder texts and it's bad for accessibility: consider adding labels above the inputt fields (should be good for mobile too)

**Comment by sionleroux** [comment]
Fixed by 3ee29bc3377775c518c63a24a616b95d950d2353
